exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -17,6 +17,7 @@
 import threading
 from abc import ABC, abstractmethod
 from collections.abc import Sequence  # noqa: F401
+from importlib.metadata import PackageNotFoundError, version
```

Comment:
Please use `importlib.metadata` in [utils](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/util/_importlib_metadata.py#L36).